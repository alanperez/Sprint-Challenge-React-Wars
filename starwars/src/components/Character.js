import React from 'react'


const Character = ({name,gender,height,mass}) => {
  // console.log('char clg')
  return(
    <div className='ui card' style={{margin: '20px auto'}}>
      <div className='content'>
        <h1>{name}</h1>
         <p><strong>gender:</strong>   {gender}</p>
         <p><strong>height:</strong>   {height}</p>
         <p><strong>mass:</strong>   {mass}</p>
      </div>

      <div className='description'>

      </div>
    </div>
  )
}

export default Character