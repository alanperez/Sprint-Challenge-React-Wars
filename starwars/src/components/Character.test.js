import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer';

import Character from './Character'

describe('<Character />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Character />).toJSON()

      expect(tree).toMatchSnapshot();
  })
})