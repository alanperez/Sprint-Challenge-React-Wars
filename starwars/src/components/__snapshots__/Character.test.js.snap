// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Character /> should match snapshot 1`] = `
<div
  className="ui card"
  style={
    Object {
      "margin": "20px auto",
    }
  }
>
  <div
    className="image"
  />
  <div
    className="content"
  />
  <div
    className="description"
  />
</div>
`;
